public/scripts/controllers/BooksIndexController.js

```diff
@@ -0,0 +1,18 @@
+angular
+	.module('bokzApp')
```

Comment:
This file uses GIANT indentation; others use 2 spaces, which is more reasonable. Again, consistency is nice.